tests/README.md

```diff
@@ -142,3 +142,23 @@ To run specific tests inside a suite:
 ```console
 _output/tests/linux_amd64/integration -test.v -test.timeout 1800s -test.run CephSmokeSuite -testify.m TestARookClusterInstallation_SmokeTest
 ```
+
+### To run tests on OpenShift environment
+
+- You have to setup OpenShift environment by your own and export KUBECONFIG before executing the tests.
+- Make sure `oc` executable file is in your PATH.
+- Currenttly we are enabling only `CephSmokeSuite`  on OpenShift environment.
+- To start the test we need to set few environment variables :-
+```
+export TEST_PLATFORM=openshift
+export TEST_ENV_NAME=openshift
+export storageClassName=gp2
```

Comment:
For consistency let's use vars with all caps
```suggestion
export TEST_STORAGE_CLASS=gp2
```

Reply:
Done.